Configure

```diff
@@ -420,6 +420,7 @@ my @disablables = (
     "asan",
     "asm",
     "async",
+    "atexit",
```

Comment:
This new option should additionally be documented in the INSTALL.md file since it applies to all platforms. I think it also justifies a CHANGES.md entry.

Reply:
Will be there on the next push.

Reply:
Updates done. Thank you for your feedback.